packages/datagateway-dataview/cypress/integration/card/dls/proposals.spec.ts

```diff
@@ -57,68 +53,7 @@ describe('DLS - Proposals Cards', () => {
       .should('exist');
   });
 
-  describe('should be able to sort by', () => {
```

Comment:
@louise-davies Do we intend to re-enable sorting one day? If so, would it be useful to keep these tests and skip over them for the time being?

Reply:
@joelvdavies has just told me @louise-davies agreed that the tests should be removed. Like Joel I just thought I'd check :) will resolve this and approve the PR

Reply:
@MRichards99 I thought that sorting is pretty much impossible due to the incompatibility of limit and order on distinct queries? Hence we would never be able to re-enable them

Reply:
@louise-davies I agree, I don't think it would be a realistic objective to get them re-enabled but I didn't know if there would be anything changing in the medium-long term to allow sorting to be fixed.